<?xml version="1.0"?>
<launch>
    <arg name="appid"               default="5e59d3a8"/>
    <arg name="chinese_awake_words" default="xiao3 bai2 xiao3 bai2"/>
    <arg name="english_awake_words" default="hello red"/>
    <arg name="language"            default="Chinese"/>

    <node pkg="xf_mic_asr_offline" type="awake_node_dds.py" name="awake_node" output="screen">
        <param name="port"      value="/dev/ttyUSB0"/>
        <param name="mic_type"  value="mic6_circle"/>
        <param if="$(eval language == 'Chinese')" name="awake_word"    value="$(arg chinese_awake_words)"/>
        <param if="$(eval language == 'English')" name="awake_word"    value="$(arg english_awake_words)"/>
    </node>
    <!--
    <node pkg="xf_mic_asr_offline" type="call_recognition" name="call_recognition" output="screen">
        <param name="confidence"        value="$(arg confidence)"/>
        <param name="seconds_per_order" value="$(arg seconds_per_order)"/>
    </node>

    <node pkg="xf_mic_asr_offline" type="voice_control" name="xf_asr_offline_node" output="screen">
        <param name="appid"                 value="$(arg appid)"/>
        <param name="confidence"            value="$(arg confidence)"/>
        <param name="seconds_per_order"     value="$(arg seconds_per_order)"/>
        <param name="source_path"           value="$(find xf_mic_asr_offline)"/>
    </node>
    -->
</launch>
